packages/react-core/src/components/Title/examples/TitleSizes.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Title, TitleSizes } from '@patternfly/react-core';
+
+export const CustomTitleSizes: React.FunctionComponent = () => (
+  <React.Fragment>
+    <Title headingLevel="h1" size={TitleSizes['4xl']}>
+      4xl Title
+    </Title>
+    <Title headingLevel="h2" size="3xl">
+      3xl Title
+    </Title>
+    <Title headingLevel="h3" size={TitleSizes['2xl']}>
+      2xl Title
+    </Title>
+    <Title headingLevel="h4" size="xl">
+      xl Title
+    </Title>
+    <Title headingLevel="h5" size={TitleSizes.lg}>
+      lg Title
+    </Title>
+    <Title headingLevel="h6" size="md">
+      md Title
+    </Title>
```

Comment:
I feel like we can better convey this example by updating the content of each title to align with the other example a bit more.

So instead of just `4xl Title`, we could instead have `h1 as 4xl`, `h2 as 3xl`, and so on.